test/MyFirstApiConsume.test.js

```diff
@@ -0,0 +1,60 @@
+const axios = require('axios');
+const { expect } = require('chai');
+const { StatusCodes } = require('http-status-codes');
+
+const path = 'https://httpbin.org';
+
+const query = {
+    name: 'Carlo',
+    age: '19',
+    city: 'New York'
+};
+
+describe('First Api Tests', () => {
+    //Returns Origin IP.
+    it('Consume GET Service', async () => {
+        const response = await axios.get(`${path}/ip`);
+        expect(response.status).to.equal(StatusCodes.OK);
+        expect(response.data).to.have.property('origin');
+    });
+
+    it('Consume GET Service with query parameters', async () => {
+        const response = await axios.get(`${path}/get`, { query });
+        expect(response.status).to.equal(StatusCodes.OK);
+        expect(response.config.query).to.eql(query);
```

Comment:
Add space between variable definition and asserts
```suggestion
        const response = await axios.get(`${path}/get`, { query });
        
        expect(response.status).to.equal(StatusCodes.OK);
        expect(response.config.query).to.eql(query);
```